package.json

```diff
@@ -107,7 +107,7 @@
   "devDependencies": {
     "@bufbuild/buf": "^1.26.1",
     "@mapeo/default-config": "4.0.0-alpha.5",
-    "@mapeo/mock-data": "^1.0.2",
+    "@mapeo/mock-data": "file:mapeo-mock-data-1.0.2.tgz",
```

Comment:
See <https://github.com/digidem/mapeo-mock-data/pull/7>. I'll deploy a new version of the dependency before merging this PR.